### PR TITLE
reopen #50354 about install_test.go of error message amending

### PR DESCRIPTION
**Pull request by jianglingxia**
**What this PR does / why we need it**:
because of unknown repository,I can not update it in before branch,so I reopen the pr #50354 
**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:
cc @irfanurrehman
**Release note**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jianglingxia** [comment]
/assign @irfanurrehman
thanks

File: federation/apis/federation/install/install_test.go [hunk 1]
```diff
@@ -36,7 +36,7 @@ func TestResourceVersioner(t *testing.T) {
 		t.Fatalf("unexpected error: %v", err)
 	}
 	if version != "10" {
-		t.Errorf("unexpected version %v", version)
+		t.Errorf("version = %v,want 10", version)
```
Comment (irfanurrehman) [inline federation/apis/federation/install/install_test.go]: s/%v,want 10/%v, want 10/

File: federation/apis/federation/install/install_test.go [hunk 2]
```diff
@@ -45,7 +45,7 @@ func TestResourceVersioner(t *testing.T) {
...
```
Comment (irfanurrehman) [inline federation/apis/federation/install/install_test.go]: s/%v,want 10/%v, want 10/

**Comment by jianglingxia** [comment]
PTAL @nikhiljindal

**Comment by irfanurrehman** [comment]
> /assign @quinton-hoole

@jianglingxia thanks for the update, meanwhile can u please update as I suggested (add space) and change the commit message to "Amend the error messages in install_test.go"; I will merge, as soon as you update this.

**Comment by jianglingxia** [comment]
thanks for your reply,and I have up updated please  review again,thanks very much! @irfanurrehman

**Pull request merged**